views/pages/resources/important-links.pug

```diff
@@ -0,0 +1,16 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+    link(href="/assets/css/pages/resources/important-links.css" rel="stylesheet")
+
+block body 
```

Comment:
Thoughts on changing up the visual style a bit? I feel like the font size is a bit big 😅 , and we can make the buttons look more consistent if we move the icons to the right!

<img width="356" alt="image" src="https://github.com/MathSoc/mathsoc-website/assets/37753525/b11ff87b-f58f-4c54-b20b-5dd18effe5ff">